hack/make.ps1

```diff
@@ -303,7 +303,10 @@ Function Run-UnitTests() {
 # Start of main code.
 Try {
     Write-Host -ForegroundColor Cyan "INFO: make.ps1 starting at $(Get-Date)"
-    $root=$(pwd)
+
+    # Get to the root of the repo
+    $root = $(Split-Path $MyInvocation.MyCommand.Definition -Parent | Split-Path -Parent)
+    Push-Location $root
```

Comment:
Does this set the actual working directory of the script? Wondering if https://github.com/docker/docker/blob/611a633ba4e11215a23d8b405a537346b10d9657/hack/make.ps1#L116-L120 still works